frontend/packages/erd-core/src/features/erd/components/ERDRenderer/CommandPalette/CommandPalettePreview/CommandPreview.tsx

```diff
@@ -0,0 +1,47 @@
+import type { StaticImageData } from 'next/image'
+import type { FC } from 'react'
+import copyLink from './assets/copy-link.mp4'
+import showAllFields from './assets/show-all-fields.png'
+import showKeyOnly from './assets/show-key-only.png'
+import showTableName from './assets/show-table-name.png'
+import tidyUp from './assets/tidy-up.mp4'
+import zoomToFit from './assets/zoom-to-fit.mp4'
+import styles from './CommandPalettePreview.module.css'
+
+type Props = {
+  commandName: string
+}
+
+const COMMAND_VIDEO_SOURCE: Record<string, string> = {
+  'copy link': copyLink,
+  'Zoom to Fit': zoomToFit,
+  'Tidy Up': tidyUp,
+}
+
+const COMMAND_IMAGE_SOURCE: Record<string, string | StaticImageData> = {
+  'Show All Fields': showAllFields,
+  'Show Key Only': showKeyOnly,
+  'Show Table Name': showTableName,
+}
```

Comment:
We may be able to define command texts type to link the `CommandOptions` and this preview,
like:

```tsx
const COMMAND_VIDEO_SOURCE: Partial<Record<CommandText, string>> = {
  'copy link': copyLink,
  'Zoom to Fit': zoomToFit,
  'Tidy Up': tidyUp,
}
```

I'll try it in another PR.